elements/rh-blockquote/rh-blockquote.ts

```diff
@@ -54,9 +54,9 @@ export class RhBlockquote extends LitElement {
   @property({ reflect: true }) size: 'default' | 'large' = 'default';
 
   render() {
-    const { on = '' } = this;
+    const { highlight, on = '' } = this;
     return html`
-      <figure id="container" class="${classMap({ [on]: !!on })}">
+      <figure id="container" class="${classMap({ highlight, [on]: !!on })}">
```

Comment:
Doesn't seem to be loading the `on` class by default

https://deploy-preview-1853--red-hat-design-system.netlify.app/elements/blockquote/demo/

DP:
![Screenshot 2024-09-13 at 2 38 05 PM](https://github.com/user-attachments/assets/0e6c688b-753f-434f-ba0c-0437e99412ed)

Expected:
![Screenshot 2024-09-13 at 2 37 59 PM](https://github.com/user-attachments/assets/b993577c-ad51-41a3-931b-780642e092ce)